alerta-covid/app/models/covid_test.rb

```diff
@@ -1,3 +1,4 @@
 class CovidTest < ApplicationRecord
   validates :name, presence: true
+  has_many  :incidence
```

Comment:
I would expect this to be plural: `incidences`, what do you think? 
> The name of the other model is pluralized when declaring a has_many association.

See: Rails Guides [2.3 The has_many association](https://guides.rubyonrails.org/association_basics.html#has-many-association-reference)